core/util.py

```diff
@@ -157,3 +224,29 @@ def custom_serialiser(obj: Any) -> str:
         return obj.isoformat()
     else:
         raise TypeError(f"Cannot serialise object of type {type(obj)}")
+
+
+def move_data_to_json_blob(
+    data: pd.DataFrame,
+    cols_to_jsonb: list[str],
+) -> pd.DataFrame:
+    """Move specified columns into a json_blob field.
+
+    This function moves specified columns in a DataFrame into a json_blob field.
```

Comment:
so can probably replace instances of "json_blob" in this doctring, as the snake case there kind of implies that "json_blob" is some kind of reference. In the docstring itself we should probably refer to is as JSONB Blob, or JSON-Binary blob storage. Something like that